docs/integrations/engines/snowflake.md

```diff
@@ -549,3 +549,13 @@ MODEL (
   ),
 );
 ```
+
+## Troubleshooting
+
+### Frequent Authentication Prompts
+
+When using Snowflake with enabled security features like Multi-Factor Authentication (MFA), you may experience repeated prompts for authentication tokens or sign-ins while running SQLMesh commands. This typically occurs when your Snowflake account is configured to restrict short-lived token issuance. To reduce authentication prompts, you can enable token caching in your Snowflake connection configuration:
+
+For general authentication, see [Connection Caching Documentation](https://docs.snowflake.com/en/user-guide/admin-security-fed-auth-use#using-connection-caching-to-minimize-the-number-of-prompts-for-authentication-optional)
```

Comment:
- For general authentication, see [Connection Caching Documentation](https://docs.snowflake.com/en/user-guide/admin-security-fed-auth-use#using-connection-caching-to-minimize-the-number-of-prompts-for-authentication-optional)

Reply:
(Start line with `- ` so it's a bullet)